index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+   <head>
+     <title>Dr. Borlaug's life</title>
+     <link rel="stylesheet" href="style.css">
+   </head>
+   <body>
+     <main id="main">
+       <h1 id="title">Dr. Norman Borlaug</h1>
+       <p>The man who saved a billion lives</p>
+       <figure id="img-div">
+    <img id="image" src="https://www.write-out-loud.com/images/commemorative-speech-candle.jpg" alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists">
+    <figcaption id="img-caption">
+    Choose the Right Synonym for tribute
+ENCOMIUM, EULOGY, PANEGYRIC, TRIBUTE, CITATION mean a formal expression of praise. ENCOMIUM implies enthusiasm and warmth in praising a person or a thing.  received encomiums from literary critics  EULOGY applies to a prepared speech or writing extolling the virtues and services of a person.  delivered the eulogy at the
+    </figcaption>
+  </figure>
+       
+     <section id="tribute-info">
+    <h3 id="headline">Here's a time line of Dr. Borlaug's life:</h3>
+    <ul>
+      <li><strong>1914</strong> - Born in Cresco, Iowa</li>
+      <li>
+        <strong>1933</strong> - Leaves his family's farm to attend the
+        University of Minnesota, thanks to a Depression era program known as the
+        "National Youth Administration"
+      </li>
+    <blockquote cite="http://news.rediff.com/report/2009/sep/14/pm-pays-tribute-to-father-of-green-revolution-borlaug.htm">
+      <p>
+        "Borlaug's life and achievement are testimony to the far-reaching
+        contribution that one man's towering intellect, persistence and
+        scientific vision can make to human peace and progress."
+      </p>
+      <cite>-- Indian Prime Minister Manmohan Singh</cite>
+    </blockquote>
+    <h3>
+      If you have time, you should read more about this incredible human being
+      on his
+      <a id="tribute-link" href="https://www.microverse.org/" target="_blank"> microverse</a>.
+    </h3>
```

Comment:
Kindly use <footer> tag instead of <h3>. This could make your code more semantic